src/fr/articles/listbox-et-navigation-clavier/index.njk

```diff
@@ -201,5 +200,3 @@ default:  // Search item starts with
 
 <p>Pour rechercher l’élément dans la liste, nous utilisons la requête <code>XPath</code> suivante&nbsp;: <code>/li/span[starts-with(text(), "la chaine à rechercher")]</code>.
```

Comment:
Is it ok to keep this line after deleting the XPath file?

Reply:
still needed